src/config/dynamic-theme-fixes.config

```diff
@@ -2836,6 +2836,19 @@ button
 
 ================================
 
+comicfury.com
+
+CSS
+#wrapper {
+    background-color: inherit !important;
+    padding-bottom: 10px !important;
+}
+#footer {
+    margin-top: 0px !important;
```

Comment:
??

Reply:
This "??" is related to the other one.

The issue is that without these two lines, a small bit of the `html` tag background still comes through when it shouldn't.

![comicfury](https://user-images.githubusercontent.com/94094391/141670576-b3439713-9e9b-4869-b197-44f1fcdcdf76.png)

The issue is is happening between `#wrapper` and `#footer`. The `#footer` has a `10px` top margin to add spacing between the two. The best way to fix this that I found was to replace the `#footer` margin with a `#wrapper` padding, as the padding uses the background color.

Reply:
Sure thing, let it be. As it's simply changing which element is adding the margin/padding.